### ISSUE TITLE
Archive::Archive defined twice

### ISSUE THREAD

**Issue by stefan6419846** [pymupdf/PyMuPDF#2036]
## Describe the bug

Building the package reports a warning:

```
swig -python -o fitz/fitz_wrap.c fitz/fitz.i
fitz/fitz.i:12342: Warning 509: Overloaded method Archive::Archive(PyObject *,char const *) effectively ignored,
fitz/fitz.i:12331: Warning 509: as it is shadowed by Archive::Archive(struct Archive *,char const *).
creating build
```

## To Reproduce

Build the wheel.

## Expected behavior

No warnings are raised.

## Your configuration

 - OpenSUSE Leap 15.3
 - PyMuPDF version 1.21.0, built from PyPI source

## Additional context

Looking into the code, both definitions are mostly identical:

https://github.com/pymupdf/PyMuPDF/blob/a730bc6d8e77c2b39586cb826d79c7c70a1030f5/fitz/fitz.i#L12321-L12342

**Comment by JorjMcKie** [comment]
Thanks for reporting this. That's no problem, please ignore.

**Comment by julian-smith-artifex-com** [comment]
Further to JorjMcKie's comment, the generation of this warning seems to be a known bug in swig.

It can be disabled with `swig_opts=['-w509']` in setup.py's call to `Extension()`; i'll push this workaround shortly.

**Comment by julian-smith-artifex-com** [comment]
Fixed in PyMuPDF-1.21.1.
